packages/loot-core/src/server/accounts/sync.js

```diff
@@ -312,7 +312,10 @@ async function normalizeNordigenTransactions(transactions, acctId) {
     }
 
     let payee_name;
-    if (trans.amount >= 0) {
+    // When the amount is equal to 0, we need to determine
+    // if this is a "Credited" or "Debited" transaction. This means
+    // that it matters whether the amount is a positive or negative zero.
+    if (trans.amount > 0 || Object.is(Number(trans.amount), 0)) {
```

Comment:
Would this work?

```suggestion
    if (Number(trans.amount) >= 0) {
```

Reply:
Nope, `Object.is` is necessary to discern between positive zero and negative zero. Your suggestion is equivalent to the existing code (`trans.amount >= 0`), just with an explicit cast rather than an implicit one.
![image](https://user-images.githubusercontent.com/6032823/224572558-69be8a46-73a3-4142-9e18-1441ce6a151a.png)